assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -617,7 +622,7 @@ export default Component.extend({
     }
   },
 
-  onScroll() {
+  onScroll(e) {
```

Comment:
That's a nitpick, but I prefer event much more than e, makes it way easier to grep for it